### PR TITLE
Drupal upgrade, removing nrwgov_base_theme, updating degov modules/fe…

### PR DESCRIPTION
**Pull request by DerH4NNES**
Drupal upgrade, removing nrwgov_base_theme, updating degov modules/features, merging lightning composer.json, removing duplicated packages

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,32 +1,17 @@
...
     "type": "project",
     "license": "GPL-2.0+",
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require-dev": {
-        "drupal/drupal-extension": "^3.2",
-        "behat/mink": "~1.7",
-        "behat/mink-goutte-driver": "~1.2",
-        "jcalderonzumba/gastonjs": "~1.0.2",
-        "drupal/coder": "8.*",
```
Comment (rawroland) [inline composer.json:13]: Did you remove all these dev dependencies, and why if you did?

File: composer.json [hunk 2]
```diff
@@ -1,32 +1,17 @@
...
-        "symfony/css-selector": "~2.8",
-        "behat/behat": "^3.0",
-        "se/selenium-server-standalone": "^2.53",
-        "composer/composer": "^1.3",
-        "drupal/twig_xdebug": "^1.0",
-        "phpdocumentor/reflection-docblock": "^3.0.0"
+        "drupal/twig_xdebug": "^1.0"
     },
     "require": {
-        "composer/installers": "^1.0",
```
Comment (rawroland) [inline composer.json]: Did you remove all these prod dependencies, and why if you did?

File: composer.json [hunk 3]
```diff
@@ -56,6 +42,10 @@
         "degov/features": {
             "type": "git",
             "url": "git@github.com:deGov/features.git"
+        },
```
Comment (rawroland) [inline composer.json]: Did you add the bower and npm asset repositories and why if you did?

File: docroot/profiles/custom/degov_profile/degov_profile.info.yml [hunk 4]
```diff
@@ -1,30 +1,28 @@
+
 name: 'deGov'
 type: profile
 distribution:
   name: deGov
   langcode: de
 description: 'Drupal 8 for Government (Drupal 8 für die öffentliche Verwaltung)'
-version: 8.x-dev
```
Comment (rawroland) [inline docroot/profiles/custom/degov_profile/degov_profile.info.yml:7]: I cannot find this tag in the documentation for the *.info.yml of a profile. Is it relevant?

**Review by rawroland — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (DerH4NNES) [reply composer.json:13]: This dependencies are also required by aquia/lightning

File: composer.json (hunk 2 above)
Comment (DerH4NNES) [reply composer.json]: This dependency is also required by aquia/lightning

File: composer.json (hunk 3 above)
Comment (DerH4NNES) [reply composer.json]: This repositoy is required by aquica/lightning

File: docroot/profiles/custom/degov_profile/degov_profile.info.yml (hunk 4 above)
Comment (DerH4NNES) [reply docroot/profiles/custom/degov_profile/degov_profile.info.yml:7]: i copied it from: https://github.com/deGov/deGov/blob/8.x-2.x-dev/docroot/profiles/custom/degov_profile/degov_profile.info.yml

**Pull request closed without merging**